judoscale-ruby/lib/judoscale/web_metrics_collector.rb

```diff
@@ -5,12 +5,6 @@
 
 module Judoscale
   class WebMetricsCollector < MetricsCollector
-    # NOTE: We collect metrics on all running web processes since they
-    # all receive and handle requests independently
-    def self.collect?(config)
-      config.current_runtime_container.web?
-    end
```

Comment:
I'm not seeing the diff where this method was actually called 🤔

Reply:
The reporter calls it on [every available metrics collector](https://github.com/judoscale/judoscale-ruby/blob/4f3dc830e04548ecd889f22b219ba3586d33967c/judoscale-ruby/lib/judoscale/reporter.rb#L27-L29). It's defined in the superclass (`MetricsCollector`), so it'll always return true for `WebMetricsCollector`.

Reply:
> It's defined in the superclass (MetricsCollector), so it'll always return true for WebMetricsCollector.

Got it, that makes sense!